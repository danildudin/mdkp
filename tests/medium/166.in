30 9 0
305 625 782 421 120 635 883 486 893 813 334 793 775 33 276 333 114 310 838 948 11 478 692 573 790 54 707 424 600 153 
975 646 170 506 770 708 525 741 154 473 424 319 875 441 998 539 480 276 435 923 260 945 735 874 995 336 414 363 573 497 
627 903 904 276 907 754 435 391 412 768 947 250 993 168 169 727 589 503 52 155 77 295 975 767 733 276 623 205 610 195 
189 597 575 250 180 666 619 870 139 822 572 431 479 410 632 882 673 237 327 757 737 382 895 10 215 63 7 51 322 551 
432 212 725 853 666 10 917 436 637 688 364 997 568 78 831 79 85 694 362 965 686 691 675 339 539 477 30 180 285 595 
34 783 248 80 422 246 722 236 374 365 988 839 164 512 741 595 76 553 711 810 777 813 89 606 550 569 321 604 40 319 
720 655 950 459 202 530 454 776 890 242 377 642 140 839 461 146 837 906 640 322 982 234 938 587 737 424 759 378 280 565 
745 643 599 837 720 444 320 814 284 107 342 919 578 991 512 924 297 24 730 24 156 897 1 384 131 313 787 829 940 151 
69 775 615 204 228 55 537 524 215 21 433 784 864 532 777 678 165 936 39 690 176 336 640 644 556 114 157 464 209 707 
787 822 68 190 944 691 766 477 769 118 876 87 503 269 803 115 247 796 786 769 842 53 250 523 163 370 998 160 287 624 
6445 5938 9116 6474 8977 6142 6374 6066 8293 
