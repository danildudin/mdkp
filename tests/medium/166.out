8675
11
5 6 8 9 11 12 18 19 24 26 28 
